src/scripts/main.js

```diff
@@ -354,7 +354,23 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const calculateAge = (born, died) => died - born;
+const calculateCentury = (year) => Math.ceil(year / 100);
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the HTML document contains an element with the class `dashboard` for the script to function correctly. If this element is missing, the script will not append any rows, and no table will be displayed.